packages/legend-query-builder/package.json

```diff
@@ -70,13 +70,15 @@
   "devDependencies": {
     "@finos/legend-dev-utils": "workspace:*",
     "@jest/globals": "29.7.0",
+    "@types/whatwg-fetch": "^0.0.33",
     "cross-env": "7.0.3",
     "eslint": "9.12.0",
     "jest": "29.7.0",
     "npm-run-all": "4.1.5",
     "rimraf": "6.0.1",
     "sass": "1.80.2",
-    "typescript": "5.6.3"
+    "typescript": "5.6.3",
+    "whatwg-fetch": "^3.6.20"
```

Comment:
I had to add `whatwg-fetch` as a `devDependency` because without it, the `Response` object cannot be created in `QueryBuilderResultPanel.test.ts`.